### DISCUSSION TITLE
Show SFTP Owner|Group as Name

### DISCUSSION THREAD

**Discussion by oabdelhalim** [iterate-ch/cyberduck#16008] [Q&A]
Hi,

How can I show the Owner/Group of FTP (SSH) file/folder as name (not number)?
The default is below.
A screenshot of a black screen

![image](https://github.com/iterate-ch/cyberduck/assets/96973575/ebc36795-edd7-4cad-83c9-d2b4294cbbab)

Thanks,

**Comment by dkocher** [comment]
Unfortunately the SFTP protocol version we support and implemented by the majority of servers does not allow for translation between the UID and GID to user and group names.

**Reply by oabdelhalim** [reply]
> Unfortunately the SFTP protocol version we support and implemented by the majority of servers does not allow for translation between the UID and GID to user and group names.

But is showing the names while using FileZilla to same servers

**Comment by sup3rgiu** [comment]
+1. Almost all popular clients show the name instead of the UID
